config/config.json

```diff
@@ -18,5 +18,6 @@
   },
   "notificationChannelID": "Notification channel ID if you set 'inNotificationChannel' to true at least once",
   "defaultEmbedColor": "#117ea6",
-  "deleteCommandMessages": true
+  "deleteCommandMessages": true,
+  "logBlocked": false
```

Comment:
`logBlocked` should be called `logBlockedUsers` and be inside the `mentionNotificator` object 👍